kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/BatchRequest.java

```diff
@@ -32,6 +32,10 @@ public class BatchRequest {
   private Map<String, String> conf = Collections.emptyMap();
   private List<String> args = Collections.emptyList();
 
+  private Map<String, String> subresourcesMap = Collections.emptyMap();
+
+  private Map<String, String> transformedSubresourcesMap = Collections.emptyMap();
```

Comment:
can we reuse the `subresourcesMap` just similar with `resource`?

Reply:
I mean `setSubresourcesMap` in server side.

Reply:
otherwise, it is confused, as it is internal but visible for customers.

Reply:
I have the same concerns. we should not leak a internal used parameter to user faced api

Reply:
Thanks for the review. `transformedSubresourcesMap` has been removed from the request DTO. Please have a look.